quarkus-pooled-jms incompatible with Quarkus 3 and Jakarta 9/10

Blocked by https://github.com/quarkiverse/quarkus-pooled-jms/issues/39

```
[INFO] Running org.apache.camel.quarkus.component.jms.artemis.it.JmsArtemisTest
[INFO] [stdout] 2023-02-06 20:25:41,520 INFO  [org.tes.doc.DockerClientProviderStrategy] (build-27) Loaded org.testcontainers.dockerclient.UnixSocketClientProviderStrategy from ~/.testcontainers.properties, will try it first
[INFO] [stdout] 2023-02-06 20:25:41,941 INFO  [org.tes.doc.DockerClientProviderStrategy] (build-27) Found Docker environment with local Unix socket (unix:///var/run/docker.sock)
[INFO] [stdout] 2023-02-06 20:25:41,943 INFO  [org.tes.DockerClientFactory] (build-27) Docker host IP address is localhost
[INFO] [stdout] 2023-02-06 20:25:41,960 INFO  [org.tes.DockerClientFactory] (build-27) Connected to docker: 
[INFO] [stdout]   Server Version: 20.10.22
[INFO] [stdout]   API Version: 1.41
[INFO] [stdout]   Operating System: Fedora Linux 35 (KDE Plasma)
[INFO] [stdout]   Total Memory: 64064 MB
[INFO] [stdout] 2023-02-06 20:25:41,969 INFO  [org.tes.uti.ImageNameSubstitutor] (build-27) Image name substitution will be performed by: DefaultImageNameSubstitutor (composite of 'ConfigurationFileImageNameSubstitutor' and 'PrefixingImageNameSubstitutor')
[INFO] [stdout] 2023-02-06 20:25:41,972 INFO  [org.tes.DockerClientFactory] (build-27) Checking the system...
[INFO] [stdout] 2023-02-06 20:25:41,972 INFO  [org.tes.DockerClientFactory] (build-27) ✔︎ Docker server version should be at least 1.6.0
[INFO] [stdout] 2023-02-06 20:25:42,027 INFO  [🐳 .io/.0.9]] (build-27) Creating container for image: quay.io/artemiscloud/activemq-artemis-broker:1.0.9
[INFO] [stdout] 2023-02-06 20:25:42,049 INFO  [🐳 .3.4]] (build-27) Creating container for image: testcontainers/ryuk:0.3.4
[INFO] [stdout] 2023-02-06 20:25:42,153 INFO  [🐳 .3.4]] (build-27) Container testcontainers/ryuk:0.3.4 is starting: d8c03271ff133e0e09309e18b5a13203f81c5828731588ec22f93f5981e6b8cd
[INFO] [stdout] 2023-02-06 20:25:42,514 INFO  [🐳 .3.4]] (build-27) Container testcontainers/ryuk:0.3.4 started in PT0.465920597S
[INFO] [stdout] 2023-02-06 20:25:42,715 INFO  [🐳 .io/.0.9]] (build-27) Container quay.io/artemiscloud/activemq-artemis-broker:1.0.9 is starting: 52f24bfc15961f5249c7fc8520f0de69a7d45d67cfac6f4096224400aaa08cb3
[INFO] [stdout] 2023-02-06 20:25:45,754 INFO  [🐳 .io/.0.9]] (build-27) Container quay.io/artemiscloud/activemq-artemis-broker:1.0.9 started in PT3.777197392S
[INFO] [stdout] 2023-02-06 20:25:45,755 INFO  [io.qua.art.cor.dep.DevServicesArtemisProcessor] (build-27) Dev Services for ActiveMQ Artemis and named configuration <default> started on tcp://localhost:32862
[INFO] [stdout] 2023-02-06 20:25:46,151 WARN  [org.apa.act.art.cor.server] (pool-2-thread-1) AMQ222018: AIO was not located on this platform, it will fall back to using pure Java NIO. If your platform is Linux, install LibAIO to enable the AIO journal
[INFO] [stdout] 2023-02-06 20:25:46,152 WARN  [org.apa.act.art.cor.server] (pool-2-thread-1) AMQ222269: Please use a fixed value for "journal-pool-files". Default changed per https://issues.apache.org/jira/browse/ARTEMIS-1628
[INFO] [stdout] 2023-02-06 20:25:46,262 INFO  [org.apa.act.art.cor.server] (pool-2-thread-1) AMQ221000: live Message Broker is starting with configuration Broker Configuration (clustered=false,journalDirectory=./target/artemis/default/journal,bindingsDirectory=./target/artemis/default/bindings,largeMessagesDirectory=./target/artemis/default/large-messages,pagingDirectory=./target/artemis/default/paging)
[INFO] [stdout] 2023-02-06 20:25:46,281 INFO  [org.apa.act.art.cor.server] (pool-2-thread-1) AMQ221013: Using NIO Journal
[INFO] [stdout] 2023-02-06 20:25:46,312 INFO  [org.apa.act.art.cor.server] (pool-2-thread-1) AMQ221057: Global Max Size is being adjusted to 1/2 of the JVM max size (-Xmx). being defined as 8,401,190,912
[INFO] [stdout] 2023-02-06 20:25:46,330 INFO  [org.apa.act.art.cor.server] (pool-2-thread-1) AMQ221043: Protocol module found: [artemis-server]. Adding protocol support for: CORE
[INFO] [stdout] 2023-02-06 20:25:46,331 INFO  [org.apa.act.art.cor.server] (pool-2-thread-1) AMQ221043: Protocol module found: [artemis-amqp-protocol]. Adding protocol support for: AMQP
[INFO] [stdout] 2023-02-06 20:25:46,331 INFO  [org.apa.act.art.cor.server] (pool-2-thread-1) AMQ221043: Protocol module found: [artemis-amqp-protocol]. Adding protocol support for: AMQP
[INFO] [stdout] 2023-02-06 20:25:46,356 INFO  [org.apa.act.aud.base] (pool-2-thread-1) AMQ601138: User anonymous@unknown is getting notification info on target resource: null []
[INFO] [stdout] 2023-02-06 20:25:46,357 INFO  [org.apa.act.aud.base] (pool-2-thread-1) AMQ601019: User anonymous@unknown is getting mbean info on target resource: org.apache.activemq.artemis.core.management.impl.ActiveMQServerControlImpl@11af69d8 []
[INFO] [stdout] 2023-02-06 20:25:46,357 INFO  [org.apa.act.aud.base] (pool-2-thread-1) AMQ601138: User anonymous@unknown is getting notification info on target resource: ActiveMQServerImpl::name=localhost []
[INFO] [stdout] 2023-02-06 20:25:46,372 INFO  [org.apa.act.art.cor.server] (pool-2-thread-1) AMQ221034: Waiting indefinitely to obtain live lock
[INFO] [stdout] 2023-02-06 20:25:46,372 INFO  [org.apa.act.art.cor.server] (pool-2-thread-1) AMQ221035: Live Server Obtained live lock
[INFO] [stdout] 2023-02-06 20:25:46,465 INFO  [org.apa.act.art.cor.server] (pool-2-thread-1) AMQ221080: Deploying address test-jms-default supporting [ANYCAST]
[INFO] [stdout] 2023-02-06 20:25:46,472 INFO  [org.apa.act.aud.base] (pool-2-thread-1) AMQ601019: User anonymous@unknown is getting mbean info on target resource: org.apache.activemq.artemis.core.management.impl.AddressControlImpl@184e8e8a []
[INFO] [stdout] 2023-02-06 20:25:46,476 INFO  [org.apa.act.art.cor.server] (pool-2-thread-1) AMQ221003: Deploying ANYCAST queue test-jms-default on address test-jms-default
[INFO] [stdout] 2023-02-06 20:25:46,511 WARN  [org.apa.act.art.cor.server] (pool-2-thread-1) AMQ222165: No Dead Letter Address configured for queue test-jms-default in AddressSettings
[INFO] [stdout] 2023-02-06 20:25:46,512 WARN  [org.apa.act.art.cor.server] (pool-2-thread-1) AMQ222166: No Expiry Address configured for queue test-jms-default in AddressSettings
[INFO] [stdout] 2023-02-06 20:25:46,528 INFO  [org.apa.act.aud.base] (pool-2-thread-1) AMQ601019: User anonymous@unknown is getting mbean info on target resource: org.apache.activemq.artemis.core.management.impl.QueueControlImpl@1514add1 []
[INFO] [stdout] 2023-02-06 20:25:46,532 INFO  [org.apa.act.aud.base] (pool-2-thread-1) AMQ601019: User anonymous@unknown is getting mbean info on target resource: org.apache.activemq.artemis.core.management.impl.AddressControlImpl@c599e2a []
[INFO] [stdout] 2023-02-06 20:25:46,564 INFO  [org.apa.act.aud.base] (pool-2-thread-1) AMQ601019: User anonymous@unknown is getting mbean info on target resource: org.apache.activemq.artemis.core.management.impl.AcceptorControlImpl@5e8e9f10 []
[INFO] [stdout] 2023-02-06 20:25:46,628 INFO  [org.apa.act.art.cor.server] (pool-2-thread-1) AMQ221020: Started EPOLL Acceptor at localhost:61616 for protocols [CORE,AMQP]
[INFO] [stdout] 2023-02-06 20:25:46,632 INFO  [org.apa.act.art.cor.server] (pool-2-thread-1) AMQ221007: Server is now live
[INFO] [stdout] 2023-02-06 20:25:46,633 INFO  [org.apa.act.art.cor.server] (pool-2-thread-1) AMQ221001: Apache ActiveMQ Artemis Message Broker version 2.26.0 [localhost, nodeID=0e1cb810-a654-11ed-ae1b-ce94b0f9cf7d] 
[INFO] [stdout] 2023-02-06 20:25:46,757 WARN  [io.qua.config] (main) Unrecognized configuration key "quarkus.https.test-port" was provided; it will be ignored; verify that the dependency extension for this configuration is set or that you did not make a typo
[INFO] [stdout] 2023-02-06 20:25:47,273 INFO  [org.apa.act.art.cor.server] (main) AMQ221002: Apache ActiveMQ Artemis Message Broker version 2.26.0 [0e1cb810-a654-11ed-ae1b-ce94b0f9cf7d] stopped, uptime 1.026 seconds
[ERROR] Tests run: 21, Failures: 0, Errors: 1, Skipped: 20, Time elapsed: 9.13 s <<< FAILURE! - in org.apache.camel.quarkus.component.jms.artemis.it.JmsArtemisTest
[ERROR] testJmsPojoConsumer  Time elapsed: 0.006 s  <<< ERROR!
java.lang.RuntimeException: java.lang.RuntimeException: Failed to start quarkus
        at io.quarkus.test.junit.QuarkusTestExtension.throwBootFailureException(QuarkusTestExtension.java:625)
        at io.quarkus.test.junit.QuarkusTestExtension.interceptTestClassConstructor(QuarkusTestExtension.java:696)
        at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.lambda$invoke$0(InterceptingExecutableInvoker.java:93)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain$InterceptedInvocation.proceed(InvocationInterceptorChain.java:106)
        at org.junit.jupiter.api.extension.InvocationInterceptor.interceptTestClassConstructor(InvocationInterceptor.java:73)
        at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.lambda$invoke$0(InterceptingExecutableInvoker.java:93)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain$InterceptedInvocation.proceed(InvocationInterceptorChain.java:106)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain.proceed(InvocationInterceptorChain.java:64)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain.chainAndInvoke(InvocationInterceptorChain.java:45)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain.invoke(InvocationInterceptorChain.java:37)
        at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.invoke(InterceptingExecutableInvoker.java:92)
        at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.invoke(InterceptingExecutableInvoker.java:62)
        at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.invokeTestClassConstructor(ClassBasedTestDescriptor.java:363)
        at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.instantiateTestClass(ClassBasedTestDescriptor.java:310)
        at org.junit.jupiter.engine.descriptor.ClassTestDescriptor.instantiateTestClass(ClassTestDescriptor.java:79)
        at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.instantiateAndPostProcessTestInstance(ClassBasedTestDescriptor.java:286)
        at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.lambda$testInstancesProvider$4(ClassBasedTestDescriptor.java:278)
        at java.base/java.util.Optional.orElseGet(Optional.java:364)
        at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.lambda$testInstancesProvider$5(ClassBasedTestDescriptor.java:277)
        at org.junit.jupiter.engine.execution.TestInstancesProvider.getTestInstances(TestInstancesProvider.java:31)
        at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$prepare$0(TestMethodTestDescriptor.java:105)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.prepare(TestMethodTestDescriptor.java:104)
        at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.prepare(TestMethodTestDescriptor.java:68)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$prepare$2(NodeTestTask.java:123)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.prepare(NodeTestTask.java:123)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:90)
        at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
        at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
        at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
        at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
        at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
        at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
        at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:35)
        at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
        at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:54)
        at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:147)
        at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:127)
        at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:90)
        at org.junit.platform.launcher.core.EngineExecutionOrchestrator.lambda$execute$0(EngineExecutionOrchestrator.java:55)
        at org.junit.platform.launcher.core.EngineExecutionOrchestrator.withInterceptedStreams(EngineExecutionOrchestrator.java:102)
        at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:54)
        at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:114)
        at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:86)
        at org.junit.platform.launcher.core.DefaultLauncherSession$DelegatingLauncher.execute(DefaultLauncherSession.java:86)
        at org.junit.platform.launcher.core.SessionPerRequestLauncher.execute(SessionPerRequestLauncher.java:53)
        at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invokeAllTests(JUnitPlatformProvider.java:150)
        at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invoke(JUnitPlatformProvider.java:124)
        at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
        at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
        at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
        at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
Caused by: java.lang.RuntimeException: Failed to start quarkus
        at io.quarkus.runner.ApplicationImpl.doStart(Unknown Source)
        at io.quarkus.runtime.Application.start(Application.java:101)
        at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
        at java.base/java.lang.reflect.Method.invoke(Method.java:578)
        at io.quarkus.runner.bootstrap.StartupActionImpl.run(StartupActionImpl.java:253)
        at io.quarkus.test.junit.QuarkusTestExtension.doJavaStart(QuarkusTestExtension.java:250)
        at io.quarkus.test.junit.QuarkusTestExtension.ensureStarted(QuarkusTestExtension.java:592)
        at io.quarkus.test.junit.QuarkusTestExtension.beforeAll(QuarkusTestExtension.java:640)
        at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.lambda$invokeBeforeAllCallbacks$12(ClassBasedTestDescriptor.java:395)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.invokeBeforeAllCallbacks(ClassBasedTestDescriptor.java:395)
        at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.before(ClassBasedTestDescriptor.java:211)
        at org.junit.jupiter.engine.descriptor.ClassBasedTestDescriptor.before(ClassBasedTestDescriptor.java:84)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:148)
        ... 36 more
Caused by: java.lang.NoClassDefFoundError: javax/jms/ConnectionFactory
        at io.quarkiverse.messaginghub.pooled.jms.PooledJmsRecorder.getWrapper(PooledJmsRecorder.java:15)
        at io.quarkus.deployment.steps.PooledJmsProcessor$wrap1501233628.deploy_0(Unknown Source)
        at io.quarkus.deployment.steps.PooledJmsProcessor$wrap1501233628.deploy(Unknown Source)
        ... 50 more
Caused by: java.lang.ClassNotFoundException: javax.jms.ConnectionFactory
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
        at io.quarkus.bootstrap.classloading.QuarkusClassLoader.loadClass(QuarkusClassLoader.java:516)
        at io.quarkus.bootstrap.classloading.QuarkusClassLoader.loadClass(QuarkusClassLoader.java:466)
        ... 53 more
```

--- a/integration-tests/pom.xml
+++ b/integration-tests/pom.xml
@@ -120,7 +120,7 @@
         <module>jdbc</module>
         <module>jfr</module>
         <!--<module>jira</module> https://github.com/apache/camel-quarkus/issues/4524 -->
-        <!--<module>jms-artemis-client</module> https://github.com/apache/camel-quarkus/issues/4500 -->
+        <module>jms-artemis-client</module>
         <module>jms-qpid-amqp-client</module>
         <module>jolt</module>
         <module>joor</module>

--- a/pom.xml
+++ b/pom.xml
@@ -57,7 +57,7 @@
         <quarkiverse-jsch.version>2.0.3</quarkiverse-jsch.version><!-- https://repo1.maven.org/maven2/io/quarkiverse/jsch/quarkus-jsch-parent/ -->
         <quarkiverse-minio.version>2.10.3</quarkiverse-minio.version><!-- https://repo1.maven.org/maven2/io/quarkiverse/minio/quarkus-minio-parent/ -->
         <quarkiverse-mybatis.version>2.0.0.CR1</quarkiverse-mybatis.version><!-- https://repo1.maven.org/maven2/io/quarkiverse/mybatis/quarkus-mybatis-parent/ -->
-        <quarkiverse-pooled-jms.version>1.0.7</quarkiverse-pooled-jms.version><!-- https://repo1.maven.org/maven2/io/quarkiverse/messaginghub/quarkus-pooled-jms-parent/ -->
+        <quarkiverse-pooled-jms.version>2.0.0.CR1</quarkiverse-pooled-jms.version><!-- https://repo1.maven.org/maven2/io/quarkiverse/messaginghub/quarkus-pooled-jms-parent/ -->
         <quarkiverse-tika.version>1.0.3</quarkiverse-tika.version><!-- https://repo1.maven.org/maven2/io/quarkiverse/tika/quarkus-tika-parent/ -->
         <quarkus.version>3.0.0.Alpha3</quarkus.version><!-- https://repo1.maven.org/maven2/io/quarkus/quarkus-bom/ -->
         <quarkus-hazelcast-client.version>3.0.0</quarkus-hazelcast-client.version><!-- https://repo1.maven.org/maven2/com/hazelcast/quarkus-hazelcast-client-bom/ -->

--- a/poms/bom/src/main/generated/flattened-full-pom.xml
+++ b/poms/bom/src/main/generated/flattened-full-pom.xml
@@ -6047,12 +6047,12 @@
       <dependency>
         <groupId>io.quarkiverse.messaginghub</groupId><!-- org.apache.camel.quarkus:camel-quarkus-bom:${project.version} -->
         <artifactId>quarkus-pooled-jms</artifactId><!-- org.apache.camel.quarkus:camel-quarkus-bom:${project.version} -->
-        <version>1.0.7</version><!-- org.apache.camel.quarkus:camel-quarkus-bom:${project.version} -->
+        <version>2.0.0.CR1</version><!-- org.apache.camel.quarkus:camel-quarkus-bom:${project.version} -->
       </dependency>
       <dependency>
         <groupId>io.quarkiverse.messaginghub</groupId><!-- org.apache.camel.quarkus:camel-quarkus-bom:${project.version} -->
         <artifactId>quarkus-pooled-jms-deployment</artifactId><!-- org.apache.camel.quarkus:camel-quarkus-bom:${project.version} -->
-        <version>1.0.7</version><!-- org.apache.camel.quarkus:camel-quarkus-bom:${project.version} -->
+        <version>2.0.0.CR1</version><!-- org.apache.camel.quarkus:camel-quarkus-bom:${project.version} -->
       </dependency>
       <dependency>
         <groupId>io.quarkiverse.minio</groupId><!-- org.apache.camel.quarkus:camel-quarkus-bom:${project.version} -->

--- a/poms/bom/src/main/generated/flattened-reduced-pom.xml
+++ b/poms/bom/src/main/generated/flattened-reduced-pom.xml
@@ -6042,12 +6042,12 @@
       <dependency>
         <groupId>io.quarkiverse.messaginghub</groupId>
         <artifactId>quarkus-pooled-jms</artifactId>
-        <version>1.0.7</version>
+        <version>2.0.0.CR1</version>
       </dependency>
       <dependency>
         <groupId>io.quarkiverse.messaginghub</groupId>
         <artifactId>quarkus-pooled-jms-deployment</artifactId>
-        <version>1.0.7</version>
+        <version>2.0.0.CR1</version>
       </dependency>
       <dependency>
         <groupId>io.quarkiverse.minio</groupId>

--- a/poms/bom/src/main/generated/flattened-reduced-verbose-pom.xml
+++ b/poms/bom/src/main/generated/flattened-reduced-verbose-pom.xml
@@ -6042,12 +6042,12 @@
       <dependency>
         <groupId>io.quarkiverse.messaginghub</groupId><!-- org.apache.camel.quarkus:camel-quarkus-bom:${project.version} -->
         <artifactId>quarkus-pooled-jms</artifactId><!-- org.apache.camel.quarkus:camel-quarkus-bom:${project.version} -->
-        <version>1.0.7</version><!-- org.apache.camel.quarkus:camel-quarkus-bom:${project.version} -->
+        <version>2.0.0.CR1</version><!-- org.apache.camel.quarkus:camel-quarkus-bom:${project.version} -->
       </dependency>
       <dependency>
         <groupId>io.quarkiverse.messaginghub</groupId><!-- org.apache.camel.quarkus:camel-quarkus-bom:${project.version} -->
         <artifactId>quarkus-pooled-jms-deployment</artifactId><!-- org.apache.camel.quarkus:camel-quarkus-bom:${project.version} -->
-        <version>1.0.7</version><!-- org.apache.camel.quarkus:camel-quarkus-bom:${project.version} -->
+        <version>2.0.0.CR1</version><!-- org.apache.camel.quarkus:camel-quarkus-bom:${project.version} -->
       </dependency>
       <dependency>
         <groupId>io.quarkiverse.minio</groupId><!-- org.apache.camel.quarkus:camel-quarkus-bom:${project.version} -->

--- a/tooling/scripts/test-categories.yaml
+++ b/tooling/scripts/test-categories.yaml
@@ -22,6 +22,7 @@ group-01:
   - google-storage
   - http
   - jcache
+  - jms-artemis-client
   - jsch
   - kafka
   - kafka-sasl

